L1Trigger/DemonstratorTools/test/gtt/createFirmwareInputFiles_cfg.py

```diff
@@ -104,11 +104,12 @@
 process.l1tTrackerEmuHTMiss.debug = (options.debug > 0)
 
 #Disable internal track selection
+#There is a problem with setting all of these (especially eta) to high numbers.
 process.l1tTrackJetsEmulation.MaxDzTrackPV = cms.double(10000.0)
 process.l1tTrackJetsEmulation.trk_zMax = cms.double(10000.0)    # maximum track z
 process.l1tTrackJetsEmulation.trk_ptMax = cms.double(10000.0)    # maximumum track pT before saturation [GeV]
 process.l1tTrackJetsEmulation.trk_ptMin = cms.double(0.0)     # minimum track pt [GeV]
-process.l1tTrackJetsEmulation.trk_etaMax = cms.double(10000.0)    # maximum track eta
+process.l1tTrackJetsEmulation.trk_etaMax = cms.double(2.4)    # maximum track eta #ETA MUST BE 2.4, this affects emulation jets
```

Comment:
```suggestion
process.l1tTrackJetsEmulation.trk_etaMax = cms.double(2.4)    # this is the max eta from the TrackWord, used in defining the eta grid edges, which must be matched
```

Reply:
This is the default actually, so it can be removed from the config like the ptMin/Max (exception: if the ptMin/Max differ from the standalone emulation/firmware TrackSelection module ...
Would be nice to add inside the TrackJetClustering source file the warning that '// this is the max eta from the TrackWord, used in defining the eta grid edges, which must be matched'